### DISCUSSION TITLE
Click and then match/replace a variable using nuclei template

### DISCUSSION THREAD

**Discussion by Sim4n6** [projectdiscovery/nuclei-templates#4886] [Q&A]
Hi, I would like to perform the following using nuclei templating: 
> in headless mode, click on a button located by XPath, but before issuing the GET request, I would like to match_replace a variable.

The current state of the template is as follows:
```
id: samples
info:
  name: samples
  author: me or not me
  
headless:
  - steps:
      - args:
          url: "{{BaseURL}}"
          redirects: true
          max-redirects: 3
        action: navigate
      - action: waitload

      - args:
          by: xpath
          xpath: /html/body/div[5]/div[3]/div/div/div/div[2]/div[1]/div[1]/button
        action: click
      - action: waitload
```

Is this possible, please?

**Comment by Sim4n6** [comment]
Clicking the button, issue a GET request to `/me?aaa=bbb` I would like to match replace a value for the variable `aaa` with multiple values `bbb, ccc, ddd, eee`. 

I hope my issue is well explained.

**Comment by MetzinAround** [comment]
Closing older issue, feel free to reopen if this is still something you want to implement.
